src/main/java/forestry/core/config/Config.java

```diff
@@ -104,6 +104,11 @@
 	private static boolean enableExUtilRedOrchid = true;
 	private static boolean enableMagicalCropsSupport = true;
 
+	// Cultivation
+	public static int planterExtend = 4;
+	public static boolean ringFarms = false;
```

Comment:
I think the farms should be rings by default, it makes them visually different from other mods and it has nostalgic appeal for a lot of players.